app/components/point-group.jsx

```diff
@@ -38,7 +38,7 @@ const PointGroup = props => {
   useEffect(() => {
     setPointObj(pointObj)
   }, [pointObj]) // could be changed by parent component, or within this component
-
+  console.log(groupedPoints, soloPoint)
```

Comment:
please remove console.log statements before final check in.
Or if there are no real changes to this file, please git checkout out master <path and file> to revert it to the original.